### DISCUSSION TITLE
How to do transparent drawing on TPaintBox32

### DISCUSSION THREAD

**Discussion by chicknsoup** [graphics32/graphics32#337] [General] [answered]
Hello,
I'm uisng TPainBox32 with TPolygonRenderer32VPR to draw charts. My question is: how to make the background of TPainBox32 transparent?

Thank you.

**Comment by andersmelander** [comment, marked as answer]
`TPaintBox32` is derived from `TCustomControl` and therefore it unfortunately doesn't support transparency against the background.
See #105 for a possible workaround.

**Reply by chicknsoup** [reply]
Thank you.
